pkg/config/config.go

```diff
@@ -65,6 +65,15 @@ var (
 	)
 )
 
+// Whitelist is the configuration fields that are visible by the `config` subcommands.
+var Whitelist = []string{
```

Comment:
Seems like we should define this in the `getWhitelist()` function, and not globally?

Reply:
I defined this globally so it could be next to the `Config` struct. That way, when modifications are made to the struct, a developer can easily add/remove a field from the whitelist. Otherwise, they would have to find the `getWhitelist` function in another directory and modify that.